### DISCUSSION TITLE
the option to overwrite an asset

### DISCUSSION THREAD

**Discussion by eswanzey** [vanjarosoftware/Vanjaro.Platform#1912] [Ideas]
If a file of the same name is uploaded in Site Assets, it is renamed to a new name that is unique so that the existing file of the same name isn't overwritten.

I'd like to see an option for "overwrite" or "add as new" before files of these types are committed to the server. And better still, the ability to "apply to all" upload items that are in the same upload queue.

**Comment by meetmandeep** [comment]
Yes. This would be a great addition.
